pkg/apis/storage/validation/validation.go

```diff
@@ -456,23 +455,6 @@ func validateStorageCapacity(storageCapacity *bool, fldPath *field.Path) field.E
 	return allErrs
 }
 
-var supportedFSGroupPolicy = sets.NewString(string(storage.ReadWriteOnceWithFSTypeFSGroupPolicy), string(storage.FileFSGroupPolicy), string(storage.NoneFSGroupPolicy))
```

Comment:
Instead of completely removing the validation, I think we need to have something more like:

- if it's already set, we don't validate it
- this field is immutable

cc @gnufied to confirm this was what we agreed on

Reply:
I can add a check to return if the field is already set, but if it isn't...

* Won't we return again? Making us just return either way without recording errors? Removing the validation previously made this function largely pointless, which is why I had removed it.
* If this isn't set, won't the fuzzer automatically set it to the default value?

Reply:
We still want to do validation on creation. It is on update where we don't want to validate the field again, so as we don't invalidate existing objects.

Reply:
I believe this is addressed as of the latest push. There is some duplication (since we no longer call the validate method during updates).